### PR TITLE
Change registration dialog

### LINKED ISSUE #2627: improve call to action in public registration modal
I'd like to suggest a few improvements to the prompts in `src/js/components/Poll/PublicRegisterModal.vue`. The current language is functionally useful, yet it seems a bit wordy and awkward to this native English speaker.

Original/existing (on the `master` branch as of commit 98fc2782df8f7a9a3e6751197c4545d59bd98c88):

01 original public register modal

Proposed:

02 updated public register modal

I suggest these tooltips:

* **Guest** - Participate in this poll as a guest without registering or logging in.
* **Name** - Provide your name or nickname to let other participants know who you are. You may include your preferred pronouns here as well. You can change your name later.
* **Email** - If you provide an email address you will receive poll reminders and a personal link to update or review your responses. You can change or remove your email address later.

### PR DESCRIPTION
**Pull request by dartcafe**
resolves #2627

### CODE REVIEW

File: src/js/components/Poll/PublicRegisterModal.vue [hunk 1]
```diff
@@ -25,67 +25,67 @@
...
+					<h2> {{ t('polls', 'Registered users') }} </h2>
 					<NcButton wide @click="login()">
 						{{ t('polls', 'Login') }}
 					</NcButton>
 					<div>
 						{{ t('polls', 'As a regular user of this site, you can participate with your internal identity after logging in.') }}
 					</div>
 					<div>
-						{{ t('polls', 'Otherwise participate publicly.') }}
+						{{ t('polls', 'Otherwise participate as guest user.') }}
```
Comment (meonkeys) [inline src/js/components/Poll/PublicRegisterModal.vue:85]: I would say "as **a** guest user"

File: src/js/components/Poll/PublicRegisterModal.vue [hunk 2]
```diff
@@ -175,90 +168,43 @@ export default {
...
 
-			return {
-				result: '',
-				status: 'success',
+		emailAddressHint() {
+			if (this.emailGeneratedStatus === 'checking') return t('polls', 'Checking email address …')
+			if (this.emailGeneratedStatus === 'mandatory') return t('polls', 'An email address is required.')
+			if (this.emailGeneratedStatus === 'invalid') return t('polls', 'Invalid email address.')
+			if (this.share.type === 'public') {
+				if (this.emailGeneratedStatus === 'valid') return t('polls', 'You will recieve your personal link after klicking "OK".')
```
Comment (meonkeys) [inline src/js/components/Poll/PublicRegisterModal.vue:194]: klicking → clicking

File: src/js/components/Poll/PublicRegisterModal.vue (hunk 1 above)
Comment (dartcafe) [reply src/js/components/Poll/PublicRegisterModal.vue:85]: yep

File: src/js/components/Poll/PublicRegisterModal.vue (hunk 2 above)
Comment (dartcafe) [reply src/js/components/Poll/PublicRegisterModal.vue:194]: omg

**Comment by meonkeys** [comment]
:+1:
